content/numpy/concepts/built-in-functions/terms/mean/mean.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.mean()'
+Description: 'Computes the arithmetic mean along the specified axis.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+## Definition
+
+The **`.mean()`** function calculates and returns the arithmetic mean, i.e. average, of an array of numbers. If the axis are specified, the average is taken over ha axis, and if not the mean is calculated across the flattened array.
+
+## Syntax
+
+```pseudo
+numpy.mean(a, axis=None, dtype=None, out=None, keepdims=<no value>, *, where=<no value>)
+```
+
+`.mean()` provides the following arguments:
+
+- `a`: It is the array of numbers for which the mean is wanted. This argument is mandatory, but all others are optional. If `a` is not a list, an error will be raised.
+- `axis`: If the axis is given, the mean is computed over the given axis or axes. Otherwise, it is calculated over the flattened array.
+- `dtype`: It is the type which used when the mean is calculated. By default, [float64](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.float64) is used for integers, and for floating point inputs, it is the same as the input `dtype`.
+- `out`: This option allows to store the calculated mean directly in an already existing array, instead of creating a new array to hold the result. Also, this optional argument must have the same shape that is expected of the resulted mean.
```

Comment:
```suggestion
- `out`: It is an optional argument that allows storing the calculated mean in an existing array instead of creating a new array to hold the result. It must have the same shape that is expected of the resulting mean. The default value for `out` is `None`.
```